### PR TITLE
Events timezone support

### PR DESCRIPTION
**Pull request by jsturek**
* Adds new plugin so moment js has timezone display functionality
* Adds timezone support to wdn events widgets.  Events will display in event's timezone.  Events with timezone different than calendar default timezone will display with timezone abbr in the time display.
* Fixed bug so `All Day` events do NOT display time

### CODE REVIEW

File: wdn/templates_5.0/js-src/events-band.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 define([
 	'wdn',
 	'jquery',
-	'moment',
+  'plugins/moment-timezone-with-data',
```
Comment (ericras) [inline wdn/templates_5.0/js-src/events-band.js]: tabs/spaces (an aside: we need a massive whitespace cleanup and reset on this project)

File: wdn/templates_5.0/js-src/events.js [hunk 2]
```diff
@@ -45,7 +53,11 @@ define([
 			}
 			var month    = '<span class="dcf-d-block dcf-txt-3xs dcf-pt-2 dcf-pb-1 dcf-uppercase dcf-bold unl-ls-2 unl-font-sans unl-cream unl-bg-scarlet">' + startDate.format('MMM') + '</span>';
 			var day      = '<span class="dcf-d-block dcf-txt-h5 dcf-bold dcf-br-1 dcf-bb-1 dcf-bl-1 dcf-br-solid dcf-bb-solid dcf-bl-solid unl-br-light-gray unl-bb-light-gray unl-bl-light-gray unl-darker-gray dcf-bg-white">' + startDate.format('D') + '</span>';
-			var time     = '<span class="dcf-d-block dcf-pt-2 dcf-txt-2xs dcf-uppercase dcf-bold unl-font-sans unl-scarlet">' + startDate.format('h:mm a') + '</span>';
+			var time     = '<span class="dcf-d-block dcf-pt-2 dcf-txt-2xs dcf-uppercase dcf-bold unl-font-sans unl-scarlet">' + startDate.format(timeformat) + '</span>';
+      if (event.DateTime.AllDay) {
+        // all day event so clear out time
+        time = '';
+      }
```
Comment (ericras) [inline wdn/templates_5.0/js-src/events.js]: whitespace

**Comment by ericras** [comment]
This ads another copy of the moment plugin.  Should the existing moment.js file be removed and this new one used in the events monthwidget ?

**Comment by jsturek** [comment]
Fixed the whitespace issues.  To ignore whitespace changes in PR diff append `?w=1` to url

**Pull request merged**